slo-reporter/overlays/zero-prod-ocp4-stage/configmap.yaml

```diff
@@ -0,0 +1,33 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: slo-reporter-stage-prod
```

Comment:
Update this in kustomize file as well.

Reply:
Moved the cronjob.yaml to resources in kustomize, what I need to update for configmap?